admin/scss/bootstrap/_variables.scss

```diff
@@ -57,7 +57,7 @@ $brand-danger:              #D40404;
 // Control the default styling of most Bootstrap elements by modifying these
 // variables. Mostly focused on spacing.
 
-$spacer:                     1rem;
+$spacer:                     1.2308rem;
```

Comment:
This is an awfully specific number. Is it magical? It might benefit from a comment as to how this number was chosen.